BrainPortal/app/views/access_profiles/_access_profiles_table.html.erb

```diff
@@ -55,7 +55,7 @@
     end
 
     t.column("Users", :users) do |s|
-      (s.users.sort_by(&:login).map { |u| link_to_user_if_accessible(u) }.join(", ").html_safe.presence) || "(None)"
+      (s.users.count.to_s.html_safe)
```

Comment:
Also, what about the list of projects in the same report? It has the same problems as the users (the list can get long...)

Maybe we show the list of users or groups when there are 4 or less, and a `...` when there are more?